scapy/arch/windows/__init__.py

```diff
@@ -472,8 +474,76 @@ def read_routes_post2008():
                            match.group(4), iface, iface.ip))
     return routes
 
+############
+### IPv6 ###
+############
+
+def in6_getifaddr():
+    """
+    Returns all IPv6 addresses found on the computer
+    """
+    ret = []
+    ps = sp.Popen([conf.prog.powershell, 'Get-NetRoute', '-AddressFamily IPV6', '|', 'select ifIndex, DestinationPrefix'], stdout = sp.PIPE, universal_newlines = True)
+    stdout, stdin = ps.communicate()
+    netstat_line = '\s+'.join(['(\d+)', ''.join(['([A-z|0-9|:]+)', '(\/\d+)'])])
+    pattern = re.compile(netstat_line)
+    for l in stdout.split('\n'):
+        match = re.search(pattern,l)
+        if match:
+            try:
+                if_index = match.group(1)
+                iface = dev_from_index(if_index)
+            except:
+                continue
+            scope = scapy.utils6.in6_getscope(match.group(2))
+            ret.append((match.group(2), scope, iface)) #(addr,scope,iface)
```

Comment:
Could you add a space after your '#' ? There are several similar fixes to apply.